smokes/e2e/hook.scenarios.ts

```diff
@@ -1,3 +1,4 @@
+import {browser, by, element, ExpectedConditions as EC} from 'protractor';
```

Comment:
Nit for prettiness. Missing a space to match other imports:
```
import { browser, by, element, ExpectedConditions as EC } from 'protractor';
```

Reply:
Fixed